modisco/tfmodisco_workflow/seqlets_to_patterns.py

```diff
@@ -420,13 +420,19 @@ def __init__(self,
 
     @classmethod
     def from_hdf5(cls, grp, track_set):
-        patterns = util.load_patterns(grp=grp["patterns"], track_set=track_set) 
-        cluster_results = None
-        total_time_taken = None
-        return cls(patterns=patterns, cluster_results=cluster_results,
-                   total_time_taken=total_time_taken)
+        success = grp.attrs["success"]
```

Comment:
Should be:
`success = grp.attrs.get("success", False)`

Reply:
Ah, makes sense, since you presumably didn't regenerate the hdf5 file. Thanks.